### ISSUE TITLE
/give command gives items in unstacked form

### ISSUE THREAD

**Issue by Mikedeejay2** [Mikedeejay2/SimpleStackPlugin#3]
It would make more sense for the give command to give items in stacked form.

**Comment by Mikedeejay2** [comment]
I don't think that this is fixable with the current Spigot API. I would need to be able to find a way to get when a player, command block, data pack, or console runs a give command which can kind of be done with the various CommandEvents from the spigot API but then I would need to get the itemstack from the give command AND the player that should be receiving those items to get the inventory of the player and then add the items through my own methods while cancelling the give command.

TLDR: I can't get the item stack from a give command, I would need to be able to get the material (which I can do) and the NBT which is currently impossible for me to get unless I don't know about a NBT String to NBT parser or I make my own (Which would be too much for such a small bug fix)
